sbin/fix_soundcard_mixer_ctrls.py

```diff
@@ -21,40 +21,40 @@
 # GNU General Public License for more details.
 #
 # For a full copy of the GNU General Public License see the LICENSE.txt file.
-# 
-#********************************************************************
+#
+# ********************************************************************
 
+import zynconf
 import os
 import sys
 
-sys.path.append(os.environ.get('ZYNTHIAN_UI_DIR',"/zynthian/zynthian-ui"))
-import zynconf
+sys.path.append(os.environ.get('ZYNTHIAN_UI_DIR', "/zynthian/zynthian-ui"))
 
-#--------------------------------------------------------------------
+# --------------------------------------------------------------------
 
 soundcard_mixer_0 = os.environ.get('SOUNDCARD_MIXER')
 
 if soundcard_mixer_0 is not None:
-	ctrls_0 = soundcard_mixer_0.split(',')
-
-	if len(ctrls_0)>0:
-		ctrls_1 = []
-		for ctrl in ctrls_0:
-			ctrl = ctrl.strip()
-			if ctrl in ("Digital","ADC","Master","Capture","DAC","Speaker","Mic","Headphone","HDMI"):
-				ctrls_1.append(ctrl+" Left")
-				ctrls_1.append(ctrl+" Right")
-			else:
-				ctrls_1.append(ctrl)
-		soundcard_mixer_1 = ",".join(ctrls_1)
-
-		# Update Config
-		if soundcard_mixer_1 != soundcard_mixer_0:
-			print("Updating SOUNDCARD_MIXER => {}".format(soundcard_mixer_1))
-			zynconf.save_config({ 
-				'SOUNDCARD_MIXER': soundcard_mixer_1
-			})
-		else:
-			print("SOUNDCARD_MIXER is OK")
-
-#--------------------------------------------------------------------
+    ctrls_0 = soundcard_mixer_0.split(',')
+
+    if len(ctrls_0) > 0:
+        ctrls_1 = []
+        for ctrl in ctrls_0:
+            ctrl = ctrl.strip()
+            if ctrl in ("Digital", "ADC", "Master", "Capture", "DAC", "Speaker", "Mic", "Headphone", "HDMI"):
```

Comment:
Does it not improve readability & concept if the Members of the list drop on a 4 space indent as a vertical list? PEP-8 demonstrates such an example but places three elements on the first row. 

in (
    "Digital",
    "ADC",
    )

Reply:
Agreed this makes sense. The initial PR performs automated PEP8 corrections. We are likely to find many instances of further format improvements like this. It may prove challenging and maybe inefficient use of resources to review all code manually. Do we address them during future code changes, i.e. update a file when touched for maintenance purpose? I could correct this one since you have identified it.

Reply:
It's a stylistic thing I tend to use as it makes cut and pasting into javascript and testing rigs easy and it's also the sort of place other people are looking for extentions and modifications.
Ideally we might propose our own stylistic guide but there are better uses of our time given the small size of the development community. As I intend to get my fingers dirty it probably makes a good starting point for low risk PRs and test rig development.